### ISSUE TITLE
Add an option to disable separators

### ISSUE THREAD

**Issue by ogarcia** [phocean/TopIcons-plus#20]
A recent commit (I think that 37763e57bdfa8df5b5415a2cf25d1a3c308f8dde) adds a HUGE separator:

![huge_topicons_separator](https://cloud.githubusercontent.com/assets/572924/15601289/c70674d2-23ee-11e6-86e3-03bb33e5a471.png)

The fasttest way to that I find to disable it is comment the following lines.

In `moveToTop()`

```
separatorLeft = addSeperator();
separatorRight = addSeperator();
```

And in `moveToTray()`

```
separatorLeft.destroy();
separatorRight.destroy();
```

With this change I have the desired result:

![normal_topicons_separator](https://cloud.githubusercontent.com/assets/572924/15601405/69180966-23ef-11e6-9e9a-950aeec1ebe2.png)

I think that you must add an option to enable-disable this separator, and (that would be great) that you can configure separator size. :wink:

**Comment by kmyi** [comment]
+1 on this one. I am doing **orgacia**'s suggestion for now but this seems like a good idea :-)

**Comment by phocean** [comment]
Thank you for your contribution. I did not notice but it is true that the separators are too big and I think it was also the cause of some rare issues (it would become suddenly very huge, forcing a shell reload).
I removed them in the dev branch.

In my opinion, I feel it is better to simply remove them, the code is cleaner, it is better integrated with other stuff in the bar and there is padding anyway.
